frontend/pages/index.tsx

```diff
@@ -66,48 +67,11 @@ const Home: NextPage = () => {
               </Typography>
               <Typography variant="body2">Please Login to continue</Typography>
             </Box>
-          ) : (
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: 'column',
-                gap: 2,
-              }}
-            >
-              {courseEvaluationListEntries.map((courseEvaluationListEntry) => (
-                <Card key={courseEvaluationListEntry.id}>
-                  <CardContent>
-                    <Box
-                      sx={{
-                        display: 'flex',
-                        flexDirection: 'row',
-                        justifyContent: 'space-between',
-                      }}
-                    >
-                      <Box>
-                        <Typography gutterBottom variant="h5" component="div">
-                          {courseEvaluationListEntry.unit_code}
-                        </Typography>
-                        <Typography variant="body2" color="text.secondary">
-                          {courseEvaluationListEntry.description}
-                        </Typography>
-                      </Box>
-                      <Box
-                        sx={{
-                          display: 'flex',
-                          alignItems: 'center',
-                        }}
-                      >
-                        <Button>View</Button>
-                      </Box>
-                    </Box>
-                  </CardContent>
-                </Card>
-              ))}
-            </Box>
-          )}
-        </CardContent>
-      </Card>
+          </CardContent>
+        </Card>
+      ) : (
+        <Listings evaluations={courseEvaluationListEntries} reviews={[]} />
```

Comment:
why not move 
```
const { response, isLoading, error } = useSWRAuth(API_ENDPOINT.COURSE_EVALUATION.LIST);
```

to the Listings component to avoid the need of props passing?